tests/job_tests/recognition/files/ref.stm

```diff
@@ -0,0 +1,5 @@
+File 1 Test 00.00 04.00 <d0> this one is correct
+File 1 Test 06.00 12.00 <d0> this one is close to correct
+File 1 Test 13.00 19.00 <d0> this one is sadly completely wrong
```

Comment:
Nitpick: This here is the ref, so it's always correct. Maybe it's easier to grasp the test if this is reflected by the ref transcription.

Reply:
Updated. Do you know why there might be the error with the last line not being recognized as comment?